EIPS/eip-4401.md

```diff
@@ -0,0 +1,115 @@
+---
+eip: 4401
```

Comment:
```suggestion
eip: 5377
```
We have a big enough problem with EIP number selection as is (and contention/debate over it), I think following a different selection mechanism here will just make the situation marginally worse.

Reply:
I mean, technically I can assign my own number here, but fine.

Reply:
*Generally* we try to have editors who are authors not act in an editorial capacity for the EIP they are authors of.  That being said, I'm not an ERC editor so one of the ERC editors could choose a different number for this (though, with the recent events I'm generally pretty apprehensive about any non-random-like number assignments happening).